src/OrleansCounterControl/CounterControl.cs

```diff
@@ -163,8 +163,10 @@ private static void InitConsoleLogging()
             Trace.Listeners.Clear();
             var cfg = new NodeConfiguration {TraceFilePattern = null, TraceToConsole = false};
             TraceLogger.Initialize(cfg);
-            var logWriter = new LogWriterToConsole(true, true); // Use compact console output & no timestamps / log message metadata
-            TraceLogger.LogConsumers.Add(logWriter);
+
```

Comment:
It just moved its implementation to a ConsoleTelemetryConsumer to avoid just boilerplate code forwarding to the new API(and to follow @sergeybykov comment https://github.com/dotnet/orleans/pull/1026#discussion-diff-45164990). The idea is to keep the public interface as is for the developer.